sample_projects/LEQuad/main_sparky_v2.cpp

```diff
@@ -109,20 +131,21 @@ int main(int argc, char** argv)
     Battery     sim_battery(sim_adc_battery);
 
     // Simulated IMU
-    Imu     sim_imu( sim.accelerometer(),
-                     sim.gyroscope(),
-                     sim.magnetometer() );
+    // Imu     sim_imu( sim.accelerometer(),
```

Comment:
you could do as it is done for megaFly_Rev4

Having all the sim related stuff below and commented so that someone can easily switch from real test to simulation